packages/lib/models/Setting.ts

```diff
@@ -1384,7 +1384,28 @@ class Setting extends BaseModel {
 				label: () => _('Ignore TLS certificate errors'),
 				storage: SettingStorage.File,
 			},
-
+			'sync.proxyUrl': {
```

Comment:
This should probably only be visible on desktop (maybe CLI too), but not on mobile where it will have no effect.